deployment/helm/human-connection/values.yaml

```diff
@@ -0,0 +1,53 @@
+# domain is the user-facing domain.
+domain: develop.human-connection.org
+# commit is the latest github commit deployed.
+commit: 889a7cdd24dda04a139b2b77d626e984d6db6781
+# dbInitialization runs the database initializations in a post-install hook.
+dbInitializion: "yarn prod:migrate init"
+# dbMigrations runs the database migrations in a post-upgrade hook.
+dbMigrations: "yarn prod:migrate up"
+# bakendImage is the docker image for the backend deployment
+backendImage: humanconnection/nitro-backend
+# maintenanceImage is the docker image for the maintenance deployment
+maintenanceImage: humanconnection/maintenance
+# neo4jImage is the docker image for the neo4j deployment
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T23:22:01Z" title="Wednesday, January 29th 2020, 12:22:01 am +01:00">Jan 29, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,31 +1,27 @@
-domain: nitro-staging.human-connection.org
+domain: develop.human-connection.org
 commit: 889a7cdd24dda04a139b2b77d626e984d6db6781
 pullPolicy: Always
 
 # configs
-developmentMailserverDomain: "helm-nitro-mailserver.human-connection.org"
-humanConnectionDomain: "helm-staging.human-connection.org"
-supportEmail: "developer@human-connection.org"
-letsencryptIssuer: "letsencrypt-staging"
+developmentMailserverDomain: "mailserver.human-connection.org"
+humanConnectionDomain: "develop.human-connection.org"
+supportEmail: "devcom@human-connection.org"
```


Do we have that email? :thinking: Or just to avoid spam?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T09:10:52Z" title="Wednesday, January 29th 2020, 10:10:52 am +01:00">Jan 29, 2020</time>_

---


this is the email that @datenbrei asked to add, but it goes back to the point I made in `Discord` that if we are going to publish our `helm` chart publicly, it would be nice to get set up with `helm-secrets` and encrypt the files.